MpsAllocatorSample/Program.cs

```diff
@@ -203,6 +203,24 @@ static async Task RequestMultiplayerServer()
             string region = Console.ReadLine();
             req2.PreferredRegions = new List<string>() {region};
             req2.SessionId = Guid.NewGuid().ToString();
+            // InitialPlayers is the list of the initial players of the game
+            // they might or might not end up connecting to the game
+            Console.WriteLine("Type initial players ID, Enter to confirm. Empty string when you're done");
+            string id = Console.ReadLine();
+            while(true)
+            {
+                if (!string.IsNullOrEmpty(id))
```

Comment:
can't this be the condition of the while loop instead of while true? might save an if else